main.go

```diff
@@ -283,7 +335,16 @@ func buildString(clientStateReply *ClientStateReply) string {
 		}
 		sb.WriteString(fmt.Sprintf("boinc_client_task_rsc_memory_bound{wuname=\"%s\",state=\"%d\"} %f\n", task.Name, task.Activetask.State, wu.RscMemoryBound))
 	}
-
+	sb.WriteString("\n")
+	sb.WriteString("# TYPE boinc_client_tasks gauge\n")
+	projects := clientStateReply.ClientState.Projects
+	wu := clientStateReply.ClientState.WorkUnits
+	apps := clientStateReply.ClientState.Apps
+	appvers := clientStateReply.ClientState.AppVersions
+	results := clientStateReply.ClientState.Results
+	for _, task := range results {
+		sb.WriteString(fmt.Sprintf("boinc_client_tasks{project=\"%s\",name=\"%s\",state=\"%d\",use=\"%s\",host=\"%s\",app=\"%s\"} %f\n", findProjectName(task.ProjectUrl, projects), task.Name, task.Activetask.State, task.Resources, host, findAppName(task.WUName, wu, appvers, apps), task.Activetask.FrictionDone*100))
```

Comment:
friction vs fraction again :)
also maybe `boinc_client_task_done_percent` (if it goes 0 - 1 or 0-100) would be better metric name?

Reply:
yes, I can change that.
I also use this for the table. so it also for only tasks.